Bugzilla/Elastic/Indexer.pm

```diff
@@ -238,7 +181,6 @@ sub _bulk_load_ids {
     }
 
     my $total = 0;
-    use Time::HiRes;
```

Comment:
Can we remove all of these at once? Why is this particular ~fish~ instance deemed worthy of zapping?